indexdata.go

```diff
@@ -236,17 +236,18 @@ func (d *indexData) memoryUse() int {
 	for _, a := range [][]uint32{
 		d.newlinesIndex, d.docSectionsIndex,
 		d.boundaries, d.fileNameIndex,
-		d.runeOffsets, d.fileNameRuneOffsets,
 		d.fileEndRunes, d.fileNameEndRunes,
 		d.fileEndSymbol, d.symbols.symKindIndex,
 	} {
 		sz += 4 * len(a)
 	}
-	sz += 8 * len(d.runeDocSections)
+	sz += d.runeOffsets.sizeBytes()
+	sz += d.fileNameRuneOffsets.sizeBytes()
+	sz += len(d.runeDocSections)
```

Comment:
Similiar comment as fileNameNagrams. This calculation is no longer correct since it needs to take into account the size of each `[]byte`. Probably better off as a `simpleSection`